Gemfile

```diff
@@ -8,7 +8,8 @@ gem 'jquery-rails'
 gem 'jbuilder'
 gem 'spring',        group: :development
 gem 'bower-rails'
-gem 'resque', github: 'resque/resque', branch: 'master'#'https://github.com/stitchfix/resque.git', branch: 'resque-redis-interface'
+# gem 'resque', github: 'resque/resque', branch: 'master'#'https://github.com/stitchfix/resque.git', branch: 'resque-redis-interface'
```

Comment:
I think you can remove this commented-out line.

History is:

* I had that branch going that makes resque-brain even possible
* They merged to master eventually, but did not do a realease, so I updated to point to master at least
* never followed up when they did do a release

Resque-brain would break horribly if they hadn't merged/released so if tests are passing with just `gem "resque"` than we're good to go.